docker/nanotube/config/config.toml

```diff
@@ -0,0 +1,53 @@
+# The settings are listed in this file are with their default values.
```

Comment:
I think these comments are applicable mostly within the `nanotube` repo.

Reply:
Just removed them